### PR TITLE
Fix travis, update dist.ini

### PR DESCRIPTION
**Pull request by autarch**

### CODE REVIEW

File: .tidyallrc [hunk 1]
```diff
@@ -1,4 +0,0 @@
-[PerlTidy]
```
Comment (oschwald) [inline .tidyallrc]: Why are we removing this?

File: tidyall.ini [hunk 2]
```diff
@@ -0,0 +1,21 @@
+[PerlTidy]
```
Comment (oschwald) [inline tidyall.ini]: Ah, I see. Will tidyall pick up on this without a flag?

Comment (autarch) [inline tidyall.ini]: Without what flag?

Comment (oschwald) [inline tidyall.ini]: `--conf-file`

Comment (autarch) [inline tidyall.ini]: Yes, tidyall looks for both .tidyallrc and tidyall.ini

**Pull request merged**